File: src/main/java/com/lucas/matches/MatchesApplication.java
package com.lucas.matches;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;

@EnableCaching
@SpringBootApplication
public class MatchesApplication {

	public static void main(String[] args) {
		SpringApplication.run(MatchesApplication.class, args);
	}

}

File: src/main/kotlin/com/lucas/matches/MatchesApplication.kt
package com.lucas.matches

import com.lucas.matches.infrastructure.util.DatabaseSeed
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cache.annotation.EnableCaching

@EnableCaching
@SpringBootApplication
class MatchesApplication

    fun main(args: Array<String>) {
        val context = SpringApplication.run(MatchesApplication::class.java, *args)
        context.getBean(DatabaseSeed::class.java).seedDatabase()
    }
